### ISSUE TITLE
New icon

### ISSUE THREAD

**Issue by Poorchop** [wallabag/firefox-ext#16]
I think this add-on could use a new icon to match the Australized icons that Firefox currently has. Something in the same shade of gray would look nice:

![untitled](https://cloud.githubusercontent.com/assets/5256208/7669151/d371b27c-fc30-11e4-91cb-52383b03a92a.png)

The wallaby is much taller than it is wide, so maybe the current "w" icon could just be modified to be a simple gray "w" and to have a clear background instead of a black background.

**Comment by Poorchop** [comment]
I'm also thinking maybe the icon can look something like this since it mostly retains the current look. This is just an example, not the actual icon that I am suggesting since I just made this in a few seconds in MS Paint. It's supposed to be a shopping bag.

![w](https://cloud.githubusercontent.com/assets/5256208/7671378/16b1f55a-fc9b-11e4-8d91-1b5ce961193b.png)

**Comment by holocronweaver** [comment]
I vote for the wallaby icon. =)

**Comment by tcitworld** [comment]
I actually like the shopping bag idea. Most icons are currently crap because I am no good with graphic tools, but you're very welcomed to propose these.

Ping @nicosomb

**Comment by nicosomb** [comment]
I like the bag idea too, but the "w" letter is too ... fat :-) and not clearly readable.

**Comment by tgrosinger** [comment]
The shopping bag is exchanging one custom icon for another. I like the suggestion from @Poorchop.

**Comment by Poorchop** [comment]
>but the "w" letter is too ... fat :-) and not clearly readable.

I'm aware, it was just something I threw together in a few minutes in the most basic image editing program, and scaling it down to 32x32 kind of messed it up. I initially assumed that someone else with more talent would make a proper icon based on my mockup, but I can try to make it myself.

@tcitworld I'm assuming you made the current toolbar icon - if you let me know which font you used for the "w", then I can make a better version of the shopping bag. Otherwise I will have to extract the "w" from an existing bitmap.

**Comment by tcitworld** [comment]
@Poorchop I used [one](https://github.com/wallabag/logo/blob/master/app-android/logo-app-android-icon-bg-black-36.png) of those : https://github.com/wallabag/logo

**Comment by Poorchop** [comment]
Here is what I came up with so far. I had a little trouble scaling down to 16x16, so the "w" is very tiny but still legible. One of the issues I noticed is that at this size, the icon almost looks like a padlock rather than a shopping bag, but I'm hoping that the flared edges at the bottom helps users to recognize it as a bag rather than a padlock.

32x32:
![mockup32](https://cloud.githubusercontent.com/assets/5256208/7717251/6d1dfef2-fe6c-11e4-8261-095b61f4bc4a.png)

16x16:
![mockup16](https://cloud.githubusercontent.com/assets/5256208/7717253/731067a0-fe6c-11e4-9b4f-e4d50b45ae78.png)

Toolbar preview (small icons):
![preview](https://cloud.githubusercontent.com/assets/5256208/7717257/7de56d1a-fe6c-11e4-8207-6d3954c65bf1.png)

If you guys are okay with this, I will submit a pull request. If you have other suggestions or ideas, I will scrap it and start over, or we can look for someone with more experience to design a nice icon. I am kind of on the fence about how this looks - the idea is okay but the execution could probably be better.

**Comment by holocronweaver** [comment]
At 32x32 it looks good, but at 16x16 it becomes hard on my eyes to make out the W. (I admittedly don't have the best vision.) I still think the icon should be more distinctive than a generic bag. With a wallaby, even if you can't make out the W, it is doubtful you will confuse the icon with anything else. Maybe a wallaby in a bag, or a bag shaped wallaby, or a wallaby shaped bag? =)

**Comment by Poorchop** [comment]
@holocronweaver I was thinking just the wallaby minus the W might look better since it would probably be more distinctive at 16x16. Removing the W from the middle of the wallaby would also mean that the wallaby wouldn't be as tall, which is a good thing since it looks nicer to have the icon as square as possible in this case.

However, the new problem would be that the wallaby would be armless, which would look kind of weird. It would be possible to etch in some arms but I'm not sure that would work at 16x16 - to look good at that size, things should be kept simple.

**Comment by holocronweaver** [comment]
@Poorchop I see what you mean. In that case I agree the wallabag bag is probably the best tradeoff between uniqueness and identifiability. A plain capital W would be clearest of all, but that may be too generic.

**Comment by Poorchop** [comment]
I think I agree with you. A plain capital W would probably be better; it's simple, legible, and works at any size.

I've also been thinking the wallaby icon over and I don't think it looks too bad at 16x16. I think that selfoss uses the same size icon and I've never felt that it looked too small there. I guess it's up to what the others think.

**Comment by nicofrand** [comment]
What about just the W which is in the current logo ? (same font etc.)

**Comment by etiess** [comment]
Very subjective, but I would vote either for a W or the bag with the W. Not the wallaby.

**Comment by jlnostr** [comment]
As @etiess, I prefer the simple W or the shopping bag. This is my version:
![logo-icon-bg-black](https://cloud.githubusercontent.com/assets/5391936/7732984/3b23ed24-ff2c-11e4-89d7-bc908e5aa896.png)

**Comment by tgrosinger** [comment]
The double lines for the handles really help make this icon look like a shopping bag. Do you have smaller sizes to compare as well?

**Comment by jlnostr** [comment]
No problem :)

16x16
![logo-icon-bg-black-16](https://cloud.githubusercontent.com/assets/5391936/7735001/961647fa-ff3a-11e4-8ded-cd1d9145f90d.png)

32x32
![logo-icon-bg-black-32](https://cloud.githubusercontent.com/assets/5391936/7735003/9b4fee24-ff3a-11e4-887b-9a643f5817d5.png)

**Comment by tgrosinger** [comment]
Definitely still works at 32, getting harder at 16 though.

**Comment by jlnostr** [comment]
It's still hard, even with little improvements.

16x16
![logo-icon-bg-black-162](https://cloud.githubusercontent.com/assets/5391936/7735206/08735120-ff3c-11e4-9436-d245ddc6d9f2.png)

16x16 with thicker lines (wow, I don't see any difference).
![logo-icon-bg-black-162](https://cloud.githubusercontent.com/assets/5391936/7735276/8d14a294-ff3c-11e4-9403-dbda7fb0b57a.png)

32x32
![logo-icon-bg-black-32](https://cloud.githubusercontent.com/assets/5391936/7735244/50ffba78-ff3c-11e4-9260-0847e364697c.png)

64x64
![logo-icon-bg-black-64](https://cloud.githubusercontent.com/assets/5391936/7735237/414f4fa8-ff3c-11e4-8734-6b21abc2c527.png)

**Comment by jlnostr** [comment]
btw, here's the code of the regarding svg file:
```
<?xml version="1.0" encoding="UTF-8" standalone="no"?>
<svg xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns="http://www.w3.org/2000/svg" enable-background="new 0 0 200 200" xml:space="preserve" viewBox="0 0 141 141" version="1.1" y="0px" x="0px" xmlns:cc="http://creativecommons.org/ns#" xmlns:dc="http://purl.org/dc/elements/1.1/"><metadata><rdf:RDF><cc:Work rdf:about=""><dc:format>image/svg+xml</dc:format><dc:type rdf:resource="http://purl.org/dc/dcmitype/StillImage"/><dc:title/></cc:Work></rdf:RDF></metadata><g transform="matrix(1.0029 0 0 1.0029 -31.872 -30.257)"><path d="m45.733 61.15h112.69l13.953 109.61h-140.59z" fill-rule="evenodd" fill="#797c80"/><g transform="matrix(1.3953 0 0 1.3953 -18.646 -92.332)" stroke="#797c80" stroke-width="3.5731" fill="none"><circle cx="78.749" cy="120.66" r="30.932"/><circle cy="120.66" cx="94.294" r="30.932"/></g><path d="m126.7 92.224c-1.6478 0-3.0152 0.49533-4.0631 1.4748-1.0604 0.99066-1.5962 2.1362-1.5962 3.4017v22.403c0 3.0054-0.533 5.2212-1.5836 6.5872-1.0158 1.32-2.6399 1.9618-4.9616 1.9618-2.3762 0-3.9947-0.63765-4.9519-1.9478-0.99902-1.366-1.5041-3.5873-1.5041-6.6011v-22.491c0-1.532-0.69764-2.785-2.0204-3.6222-1.7985-1.133-3.5859-1.5418-5.7612-0.81903-0.66416 0.22185-1.2893 0.51067-1.8599 0.8581-0.60696 0.36975-1.119 0.83299-1.5251 1.3744-0.44928 0.60276-0.67811 1.2572-0.67811 1.9478v22.753c0 3.011-0.50649 5.2323-1.5055 6.5997-0.95996 1.3102-2.5799 1.9478-4.9505 1.9478-2.379 0-4.0142-0.64044-5.0007-1.9548-1.0241-1.366-1.5446-3.5859-1.5446-6.5941v-22.403c0-1.2669-0.53579-2.4111-1.5976-3.4017-1.0479-0.97949-2.4432-1.4748-4.1482-1.4748-1.7553 0-3.2036 0.49114-4.3045 1.4581-1.1316 0.99065-1.705 2.1418-1.705 3.4185v22.054c0 2.7738 0.27069 5.3984 0.80229 7.7969 0.54835 2.4529 1.5027 4.6044 2.8394 6.3988 1.3506 1.8125 3.1841 3.2412 5.45 4.2556 2.2422 0.99903 5.0468 1.5055 8.3369 1.5055 3.4282 0 6.2997-0.63486 8.5322-1.8892 1.8725-1.052 3.4506-2.4334 4.7035-4.1175 1.2083 1.6841 2.7501 3.0655 4.5961 4.1161 2.2046 1.2544 5.1193 1.8892 8.6661 1.8892 3.2887 0 6.0821-0.50788 8.2978-1.5083 2.2339-1.013 4.0491-2.4432 5.4026-4.2528 1.3395-1.7999 2.2952-3.9529 2.838-6.3988 0.5316-2.3985 0.80229-5.023 0.80229-7.7969v-22.053c0-1.2711-0.55533-2.418-1.652-3.4115-1.0702-0.97251-2.5366-1.465-4.3533-1.465z" fill="#fff"/></g></svg>

```

**Comment by Poorchop** [comment]
I agree with @tgrosinger that the double lines for the handles look nicer but I think they need to be thicker to look well at 16x16.  I think that you should also make the W transparent in order to follow the australis style.

To test in the Firefox toolbar, just clone this repo and replace the current icons in the [skin directory](https://github.com/wallabag/firefox-ext/tree/master/chrome/skin) with your custom icons. Then create a .zip file with `chrome`, `chrome.manifest`, and `install.rdf` at the root of the .zip file, change the extension from .zip to .xpi, and then click and drag into Firefox.

**Comment by jlnostr** [comment]
Better?
![unbenannt](https://cloud.githubusercontent.com/assets/5391936/7742065/e522f43e-ff89-11e4-9814-2b4be9db91ba.PNG)
16x16: ![wallabag-shopping-bag-48](https://cloud.githubusercontent.com/assets/5391936/7742082/3d46b164-ff8a-11e4-8b89-310198f12d01.png)


I also tried it with just the W. I prefer it because of icon consistency.
![unbenannt2](https://cloud.githubusercontent.com/assets/5391936/7742071/019ff18e-ff8a-11e4-9956-7916854ba1c4.PNG)
16x16: ![toolbar-small](https://cloud.githubusercontent.com/assets/5391936/7742077/187062e0-ff8a-11e4-8dcb-4538747e4a84.png)

**Comment by Poorchop** [comment]
Yeah the bag icon looks better but I think I prefer the basic W too because as you mentioned, it's more consistent and I think the bolder strokes go a long way. The bag icon looks more interesting while the W looks more professional. I vote for the simple W but I'd be happy with either icon.

**Comment by holocronweaver** [comment]
I too think the plain W looks best, especially at 16x16.

**Comment by N07070** [comment]
Woot for the plain W.

**Comment by stepardo** [comment]
I also like that 'plain W'. It might look strange/distracting when put into the context menu, though.
